packages/common/e2e/helpers/Artifactory.ts

```diff
@@ -98,7 +101,7 @@ export default class Artifactory {
     return this.getOrg(name).fixtures as Record<string, unknown>;
   }
 
-  getFixtureId(org: string, path: string): string {
-    return getProp(this.getOrg(org).fixtures, path);
-  }
+  // getFixtureId(org: string, path: string): string {
```

Comment:
Do we plan on re-implementing this? Just want to avoid lots of stray comments

Reply:
This is just test harness code - I can remove, but it's non-shipping